node/src/rpc.rs

```diff
@@ -7,16 +7,32 @@
 
 use std::sync::Arc;
 
-use creditcoin_node_runtime::{opaque::Block, AccountId, Balance, Index};
+use creditcoin_node_runtime::{opaque::Block, AccountId, Balance, BlockNumber, Hash, Index};
 use jsonrpsee::RpcModule;
+use sc_consensus_grandpa::FinalityProofProvider;
+use sc_consensus_grandpa_rpc::{Grandpa, GrandpaApiServer};
 pub use sc_rpc_api::DenyUnsafe;
 use sc_transaction_pool_api::TransactionPool;
 use sp_api::ProvideRuntimeApi;
 use sp_block_builder::BlockBuilder;
 use sp_blockchain::{Error as BlockChainError, HeaderBackend, HeaderMetadata};
 
+/// Dependencies for GRANDPA
+pub struct GrandpaDeps<B> {
+	/// Voting round info.
+	pub shared_voter_state: sc_consensus_grandpa::SharedVoterState,
+	/// Authority set info.
+	pub shared_authority_set: sc_consensus_grandpa::SharedAuthoritySet<Hash, BlockNumber>,
+	/// Receives notifications about justification events from Grandpa.
+	pub justification_stream: sc_consensus_grandpa::GrandpaJustificationStream<Block>,
+	/// Executor to drive the subscription manager in the Grandpa RPC handler.
+	pub subscription_executor: sc_rpc::SubscriptionTaskExecutor,
+	/// Finality proof provider.
+	pub finality_provider: Arc<FinalityProofProvider<B, Block>>,
+}
```

Comment:
I think you can add either a unit test and/or an integration test to query this RPC and make sure it returns something. Not sure what the assertion would be, maybe just don't crash and return non empty response.